query-engine/connectors/sql-query-connector/src/error.rs

```diff
@@ -313,6 +321,7 @@ impl From<quaint::error::Error> for SqlError {
             e @ QuaintKind::PoolTimeout { .. } => SqlError::ConnectionError(e),
             e @ QuaintKind::PoolClosed { .. } => SqlError::ConnectionError(e),
             e @ QuaintKind::TlsError { .. } => Self::ConnectionError(e),
+            QuaintKind::TransactionWriteConflict => todo!(),
```

Comment:
We should be sure the `todo` is not released, and the conflict error is handled correctly. Please communicate with @SevInf!

Reply:
Of course, I had already noticed that @SevInf had published to the main branch of quaint last